src/kicker/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ set(CMAKE_CXX_COMPILER  ${AVR_CXX_COMPILER})
 # change above to match ATTINY84A =====================================================================
 # to keep timing for neopixel
 # The -O2 turns on optimizations, which are required to use the delay_*() functions
-list(APPEND CMAKE_C_FLAGS "-std=c11 -O2 -mmcu=attiny167 -DF_CPU=1000000")
```

Comment:
Just wondering. Why did this change?

Reply:
As far as we can tell it was never correct. It doesn't actually affect the clock or core timer scalers, just some delay/wait functions.